libraries/src/Helper/MediaHelper.php

```diff
@@ -24,6 +26,14 @@
  */
 class MediaHelper
 {
+	/**
+	 * Responsive image size options
+	 *
+	 * @var    array
+	 * @since  4.1.0
```

Comment:
```suggestion
	 * @since  __DEPLOY_VERSION__
```

Please make this change everywhere. We have build scripts that will replaced __DEPLOY_VERSION__ with the actual version. It might not make it into 4.1.0 for all sorts of reasons so this saves everyone time.